### DISCUSSION TITLE
Connection taking long between 2x Win 11 PCs since 24H2 update

### DISCUSSION THREAD

**Discussion by kingmustard** [rustdesk/rustdesk#9580] [General]
### Bug Description

I have two PCs in the house, both with Windows 11 and both updated to the new 24H2.

Since the updating to 24H2, connections between the systems can take minutes (even though RustDesk itself is showing Green and appears connected to the Internet), and often, failing to connect at all.

This started happening instantly after updating to 24H2, if it helps.

### How to Reproduce

Happens every time.

### Expected Behavior

Fast connection.

### Operating system(s) on local side and remote side

Windows 11 24H2 local and remote

### RustDesk Version(s) on local side and remote side

1.2.6

### Screenshots

No screenshot required; generally either spins forever or shows Timeout.

**Comment by geedeee** [comment]
I can confirm that this is also a problem in RustDesk V1.3.1 since updating Windows 11 to 24H2 from 23H2. Connection is happening over a switched gigabit LAN. 
Prior to this problem, the Windows 11 'Client' was on 24H2 and the Windows 11 'Server' was on 23H2, with no problems at all. Problem occurred after upgrading the 'Server' to 24H2.
Very slow laggy connection or unable to connect at all.

**Reply by geedeee** [reply]
Ok,
I uninstalled the RustDesk software from the Windows 11 'Server' 24H2
rebooted
deleted %USERPROFILE%\AppData\Local\RustDesk
deleted %USERPROFILE%\AppData\Roaming\RustDesk
rebooted
Downloaded the latest 'nightly' build which was ' rustdesk-1.3.2-x86_64.msi ' dated 10th Oct 2024.
Installed this 'nightly' build on the Windows 11 'Server' 24H2
rebooted
Attempt to connect from Windows 11 'Client' 24H2 to Windows 11 'Server' 24H2 -> Success!
Connect fast, first time. No lag with dragging windows, desktop interaction etc. File transfers & copying back to 'normal' speeds.
HTH somebody.

**Reply by geedeee** [reply]
I forgot to add, that after reinstallation, I did not need to reconfigure the Windows 11 24H2 'Server' at all.
